x-pack/test/alerting_api_integration/spaces_only/tests/alerting/event_log.ts

```diff
@@ -411,16 +414,19 @@ export function validateEvent(event: IValidatedEvent, params: ValidateEventLogPa
   if (duration !== undefined) {
     expect(typeof duration).to.be('number');
     expect(eventStart).to.be.ok();
-    expect(eventEnd).to.be.ok();
 
-    const durationDiff = Math.abs(
-      Math.round(duration! / NANOS_IN_MILLIS) - (eventEnd - eventStart)
-    );
+    if (shouldHaveEventEnd !== false) {
```

Comment:
would it make sense to have a check when `shouldHaveEventEnd == false`, so that if there is an end event when we don't expect one, it would flag as an error

Reply:
Good call. Updated in this commit: https://github.com/elastic/kibana/pull/101387/commits/1b6a4c0bd6da407d11d2eff5e38bb3d085c027a7